src/app/pages/MarginTradePage/components/NotificationSettingsDialog/index.tsx

```diff
@@ -76,6 +85,33 @@ export const NotificationSettingsDialog: React.FC<INotificationSettingsDialogPro
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [notificationUser]);
 
+  const updateSubscriptions = () => {
```

Comment:
Wrap this function in `useCallback()` with `notificationUser` in dependency array

Reply:
I don't think the result of this should be cached - this function is only called when the user actually submits the form. If notificationUser is the dependency array, then when the user submits the form the notificationUser is not updated and the form shows the old version

Reply:
Thanks soulbit! Making this change now